test/e2e/adot_test.go

```diff
@@ -0,0 +1,33 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"testing"
+
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/test/framework"
+)
+
+func TestCPackagesAdotInstallSimpleFlow(t *testing.T) {
```

Comment:
I totally understand the premise of starting small on the tests, but the naming convention here should include the K8s version number, and provider. I think at a minimum you should be running one test on each provider as well.

Reply:
usually it's k8s version, Node OS, Provider in the name. It's a bit cumbersome, but it's nice to see what is failing just by looking at a test name, rather than digging into the code.
`TestCPackagesEmissaryDockerUbuntuKubernetes122SimpleFlow`

Reply:
Added!